index.ts

```diff
@@ -1,21 +1,39 @@
 import * as bunyan from 'bunyan';
 
+const SANITIZED_KEYS = ['password', 'secret', 'credentials'];
 
 let rootLogger = bunyan.createLogger({
     name: process.env.APP_NAME || "app",
     src: !(process.env.BUNYAN_NO_SRC),
     level: (process.env.BUNYAN_LEVEL || 'info'),
     serializers: {
-        err: err => Object.assign({
-            message: err.message,
-            name: err.name,
-            stack: getFullErrorStack(err),
-            code: err.code,
-            signal: err.signal
-        }, err)
+        err: err => {
+            sanitizeObject(err);
+            return Object.assign({
+                message: err.message,
+                name: err.name,
+                stack: getFullErrorStack(err),
+                code: err.code,
+                signal: err.signal
+            }, err)
+        }
     }
 });
 
+function sanitizeObject(input: { [key: string]: any }) {
```

Comment:
could we instead have sanitizeObject return a new object  that we assign onto the old one?
e.g. returns `{ stupid_password: 'SANITIZED' } `